datasetComparison/src/main/resources/reference.conf

```diff
@@ -4,4 +4,6 @@ dataset-comparison {
   comparisonUniqueId = "ComparisonUniqueId"
   actualPrefix = "actual"
   expectedPrefix = "expected"
+  allowDuplicates = false
+  deduplicate = true
```

Comment:
These should be explained somewhere like the `README`, they seem a little unintuitive to me. I think the confusion comes from the fact that `allow duplicates` means `ignore duplicates on read`, while `deduplicate` means `ignore duplicates on write`. Which is still a confusing distinction... I might be wrong in my interpretation.

Reply:
This will be in GitHub pages. I don't want to bloat the readme. Is that ok?